configs/projects/puppet-agent.rb

```diff
@@ -241,7 +241,14 @@
   end
   proj.component "ruby-shadow" unless platform.is_aix? || platform.is_windows?
   proj.component "ruby-augeas" unless platform.is_windows?
-  proj.component "openssl"
+
+  if ENV["USE_SYSTEM_OPENSSL"]
```

Comment:
One of the prior versions I submitted explicitly checked the environment variable's value for "1" but had changed it per some earlier suggestion/comments then.